app_noticias/templates/app_noticias/detalhes.html

```diff
@@ -1,11 +1,12 @@
-<!DOCTYPE html>
-<html lang="pt-br">
-<head>
-    <meta charset="UTF-8">
-    <title>{{ noticia.titulo }}</title>
-</head>
-<body>
+{% extends "app_noticias/base.html" %}
+
+{% block titulo %}{{noticia.titulo}}{% endblock titulo %}
+
+{% block conteudo %}
 <h1>{{ noticia.titulo }}</h1>
+<div>
+    <img src="\media\117080_original_5184x3456.jpg" class="img-responsive">
```

Comment:
deveria apresentar uma foto da notícia, não uma foto definida diretamente no código